shopinvader_product_attribute_set/models/product_filter.py

```diff
@@ -10,7 +10,8 @@ class ProductFilter(models.Model):
     _inherit = "product.filter"
 
     based_on = fields.Selection(
-        selection_add=[("product_attribute", "Product Attribute")]
+        selection_add=[("product_attribute", "Product Attribute")],
+        ondelete={"product_attribute": "cascade"},
```

Comment:
is v12 compatible ?

Reply:
good question, it is mandatory since V14, I don't know before, I'll check it.

Reply:
I deleted it